pandas/core/categorical.py

```diff
@@ -1862,8 +1990,11 @@ def convert_categorical(x):
     if get_dtype_kinds(to_concat) - set(['object', 'category']):
         # convert to object type and perform a regular concat
         from pandas.core.common import _concat_compat
-        return _concat_compat([np.array(x, copy=False, dtype=object)
-                               for x in to_concat], axis=0)
```

Comment:
Agree with @hayd on patching things after the facts. I would say, or we use YAPF and take it as it is (maybe with a few exceptions), or if we don't like the output just don't use it. 
Because now it is creating a huge diff, which seems much work to edit all these manually.

I tried to play with some of the config, but as far as I could find, I didn't directly see an option to change this behaviour ..